docs: add more information about `onCancel` property (#4659)
refinedev/refine: .changeset/forty-months-hang.md

---
"@refinedev/core": patch
---

> chore: add tsdoc to properties on following hooks:

Reviewer: ```suggestion
chore: fix tsdoc description of `onCancel` property on following hooks:
```
